docs/guided-remediation.md

```diff
@@ -0,0 +1,247 @@
+---
+layout: page
+title: Guided Remediation
+permalink: /experimental/guided-remediation/
+parent: Experimental Features
+nav_order: 3
+---
+
+# Guided Remediation
+
+Experimental
+{: .label }
+
+{: .no_toc }
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+- TOC
+{:toc}
+</details>
+
+The `osv-scanner fix` subcommand leverages [deps.dev](https://deps.dev) to provide automated and guided remediation of vulnerabilities in your project's dependencies by suggesting upgrades to dependencies.
```

Comment:
I'd add a paragraph before this to describe the motivation of this and to define "guided remediation".

Something along the lines of:

```
Guided remediation aims to help developers with fixing with the high number of known vulnerabilities in dependencies typically reported by vulnerability scanners by providing a small number of actionable steps. 

This tool provides several options to users for how to prioritise and remediate their vulnerabilities, with easy to understand information on how we arrived at these options and how to compare them. This includes features such as:

- Resolution and analysis of the entire transitive graph to determine the minimal changes required to remove vulnerabilities.
- Prioritising direct dependency upgrades by the total number of transitive vulnerabilities fixed. 
- Prioritising vulnerabilities by dependency depth, severity, and whether or not to care about dev-only dependencies.
- Applying different strategies with different risk/reward ratios (e.g. in-place fixes vs relocking).
```

WDYT?

Reply:
I think this is good - I've interspersed it with the existing paragraph.